### ISSUE TITLE
multiple people try to run console on same app

### ISSUE THREAD

**Issue by lnguyen** [danhigham/cf-console#2]
when multiple people try to run console on same app it adds too many commands and causes problems. It looks like its just append command with more commands > Updating app start command to 'curl http://tmate-bootstrap.cfapps.io?cmd=curl+http%3A%2F%2Ftmate-bootstrap.cfapps.io%3Fcmd=rackup+-p+8080+%7C+sh | sh'.

**Comment by danhigham** [comment]
Yup, there will be some logic to check if there is an app instance already running a tmate client. I wrote this two days ago and still have plenty to do! Thanks for the heads up though :-)

**Comment by lnguyen** [comment]
awesome thanks for writing this
